intg-androidtv/external_metadata.py

```diff
@@ -0,0 +1,84 @@
+import base64
+import json
+import logging
+import os
+from io import BytesIO
+from pathlib import Path
+from typing import Dict
+
+import google_play_scraper
+import requests
+from PIL import Image
```

Comment:
PIL is missing, runtime exception

Reply:
Yep already corrected. For got to include the requirements.txt in my latest commit.